doc/modules/mixture.rst

```diff
@@ -211,49 +211,73 @@ Cons
 
-.. |plot_gmm_sin| image:: ../auto_examples/mixture/images/sphx_glr_plot_gmm_sin_001.png
+.. figure:: ../auto_examples/mixture/images/sphx_glr_plot_gmm_sin_001.png
    :target: ../auto_examples/mixture/plot_gmm_sin.html
-   :scale: 48%
+   :align: center
+   :scale: 65%
 
-.. centered:: |plot_gmm| |plot_gmm_sin|
+On that example, we are fitting a dataset not well-depicted by a Gaussian
+mixture. Adjusting the `beta_concentration_prior`, parameter of the DPGMM
+controls the number of components used to fit this data.
 
+.. figure:: ../auto_examples/mixture/images/sphx_glr_plot_dirichlet_process_mixture_001.png
+   :target: ../auto_examples/mixture/plot_dirichlet_process_mixture.html
+   :align: center
+   :scale: 50%
 
-The examples above compare Gaussian mixture models with fixed number of
-components, to DPGMM models. **On the left** the GMM is fitted with 5
-components on a dataset composed of 2 clusters. We can see that the DPGMM is
-able to limit itself to only 2 components whereas the GMM fits the data fit too
-many components. Note that with very little observations, the DPGMM can take a
-conservative stand, and fit only one component. **On the right** we are fitting
-a dataset not well-depicted by a Gaussian mixture. Adjusting the `alpha`
-parameter of the DPGMM controls the number of components used to fit this
-data.
+The previous figure presents the resulting clusters computed by the Dirichlet
+Gaussian mixtures for different values of `beta_concentration_prior`. The
+`beta_concentration_prior` is directly linked to the number of clusters
+obtained. As for the Variational Bayesian Gaussian Mixtures, small value of this
+parameters will lead to some mixture components while high value leads more
+mixture components but in a more stable way (each component is activated only if
+it is necessary).
 
 .. topic:: Examples:
 
-    * See :ref:`sphx_glr_auto_examples_mixture_plot_gmm.py` for an example on plotting the
-      confidence ellipsoids for both :class:`GaussianMixture`
-      and :class:`DPGMM`.
+    * See :ref:`sphx_glr_auto_examples_mixture_plot_gmm.py` for an example on
+      plotting the confidence ellipsoids for both :class:`GaussianMixture`
+      and :class:`DirichletGaussianMixture`.
 
     * :ref:`sphx_glr_auto_examples_mixture_plot_gmm_sin.py` shows using
-      :class:`GaussianMixture` and :class:`DPGMM` to fit a sine wave
+      :class:`GaussianMixture` and :class:`DirichletGaussianMixture` to fit a
+      sine wave.
+
+    * :ref:`sphx_glr_auto_example_mixture_plot_dirichlet_process_mixture.py`
+      for an example plotting the confidence ellipsoids for the
+      :class:`DirichletGaussianMixture` for different values of the parameter
+      `beta_concentration_prior`.
+
 
-Pros and cons of class :class:`DPGMM`: Dirichlet process mixture model
-----------------------------------------------------------------------
+Pros and cons of class :class:`DirichletGaussianMixture`: Dirichlet process mixture model
```

Comment:
It's more `Pros and cons of Dirichlet process with class :class:DirichletGaussianMixture`